plugins/SitesManager/API.php

```diff
@@ -394,6 +394,22 @@ public function getSitesWithAdminAccess($fetchAliasUrls = false, $pattern = fals
         return $sites;
     }
 
+    /**
+     * Returns the messages to warn users on site deletion.
+     *
+     * @throws Exception if the website ID doesn't exist or the user doesn't have access to it
+     * @param int $idSite
+     * @return string messages to warn users
+     */
+    public function getMessagesToWarnOnSiteRemoval($idSite)
+    {
+        $messages = [Piwik::translate('SitesManager_DeleteSiteExplanation')];
```

Comment:
The initial message should not be part of that array, but should be hard coded in the vue code. Only additional messages should be provided here.

Reply:
Removed